test/squish/suite_VerifySettingsDialogue/tst_VerifySettingsDialogue/test.py

```diff
@@ -0,0 +1,153 @@
+    # verify correct storage profile name is set
+    test.compare(
+        str(squish.waitForObjectExists(gui_locators.farm_settings_QComboBox_2).currentText),
+        storage_profile,
+        "Expect selected storage profile to be set.",
+    )
+    # set job attachments filesystem options
+    gui_helpers.set_job_attachments_filesystem_options(job_attachments)
+    # verify job attachments filesystem options is set to 'COPIED'
+    test.compare(
+        str(squish.waitForObjectExists(gui_locators.farm_settings_QComboBox_3).currentText),
+        job_attachments,
+        "Expect selected job attachment filesystem option to be set.",
+    )
+    # verify 'COPIED' contains correct tooltip text
+    test.compare(
+        str(squish.waitForObjectExists(gui_locators.farm_settings_QComboBox_3).toolTip),
+        tooltip_text_copied,
+        "Expect COPIED to contain correct tooltip text.",
+    )
+    # verify job attachments filesystem options lightbulb icon contains correct tooltip text
+    test.compare(
+        str(
+            squish.waitForObjectExists(
+                gui_locators.farm_settings_Job_attachments_filesystem_options_QLabel_2
+            ).toolTip
+        ),
+        tooltip_text_lightbulb,
+        "Expect job attachments filesystem options lightbulb icon to contain correct tooltip text.",
+    )
+    # verify auto accept prompt defaults checkbox is checkable
+    test.compare(
+        squish.waitForObjectExists(
+            gui_locators.general_settings_Auto_accept_prompt_defaults_QCheckBox
+        ).checkable,
+        True,
+        "Expect auto accept prompt defaults checkbox to be checkable.",
+    )
+    # verify telemetry opt out checkbox is checkable
+    test.compare(
+        squish.waitForObjectExists(
+            gui_locators.general_settings_Telemetry_opt_out_QCheckBox
+        ).checkable,
+        True,
+        "Expect telemetry opt out checkbox to be checkable.",
+    )
+    # set conflict resolution option
+    gui_helpers.set_conflict_resolution_option(conflict_res_option)
+    # verify conflict resolution option is set to 'NOT_SELECTED'
+    test.compare(
+        str(
+            squish.waitForObjectExists(
+                gui_locators.general_settings_Conflict_resolution_option_QComboBox
+            ).currentText
+        ),
+        conflict_res_option_expected_text,
+        "Expect selected conflict resolution option to be set.",
+    )
+    # set current logging level option
+    gui_helpers.set_current_logging_level(logging_level)
+    # verify current logging level option is set to 'WARNING'
+    test.compare(
+        str(
+            squish.waitForObjectExists(
+                gui_locators.general_settings_Current_logging_level_QComboBox
+            ).currentText
+        ),
+        logging_level,
+        "Expect selected current logging level to be set.",
+    )
+    # disabling due to https://sim.amazon.com/issues/Bea-28289
```

Comment:
remove link

Reply:
Thanks. In previous automated tests, we added the bug link as it was helpful for QAEs (and developers) to reference the issue number, and keep track of things as they get fixed so tests can then be updated (an example is here: https://code.amazon.com/packages/MadStudioE2ETests/blobs/b42d2505354c9523b3b9c420dbbaeabe8d818608/--/src/stop-reconnect-spec.ts#L78). I assume leaving the ticket number is still OK?

Reply:
Update - I spoke with Marshall. As this is an open source repo, we wouldn't want to leave any confidential information in the code. So will go ahead and remove the bug ticket numbers.